python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -67,7 +67,44 @@ def to_fivetran_connector_table_props_data(self) -> Sequence[FivetranConnectorTa
         """Method that converts a `FivetranWorkspaceData` object
         to a collection of `FivetranConnectorTableProps` objects.
         """
-        raise NotImplementedError()
+        data: List[FivetranConnectorTableProps] = []
+
+        for connector_id, connector_data in self.connectors_by_id.items():
+            connector_details = connector_data.properties
+            connector_name = connector_details["schema"]
+            connector_url = get_fivetran_connector_url(connector_details)
+
+            schema_config = connector_details["schema_config"]
+
+            destination_details = self.destinations_by_id[
+                connector_details["destination_id"]
+            ].properties
+            database = destination_details.get("config", {}).get("database")
+            service = destination_details.get("service")
+
+            schemas_data = cast(Dict[str, Any], schema_config["schemas"])
+            for schema in schemas_data.values():
+                if schema["enabled"]:
+                    schema_name = schema["name_in_destination"]
+                    schema_tables: Dict[str, Dict[str, Any]] = cast(
+                        Dict[str, Dict[str, Any]], schema["tables"]
+                    )
+                    for table in schema_tables.values():
+                        if table["enabled"]:
+                            table_name = table["name_in_destination"]
+                            data.append(
+                                FivetranConnectorTableProps(
+                                    table=f"{schema_name}.{table_name}",
```

Comment:
This feels like it should be a standalone method / something reusable so that we're consistent about how it's constructed.

Reply:
Done in [0040d59](https://github.com/dagster-io/dagster/pull/25797/commits/0040d595a8a47490cee73d32211b8e5939e89a98)